ddl/column.go

```diff
@@ -304,7 +304,8 @@ func (d *ddl) addTableColumn(t table.Table, columnInfo *model.ColumnInfo, reorgI
 	for {
 		startTime := time.Now()
 		handles = handles[:0]
-		err = iterateSnapshotRows(d.store, t, version, seekHandle,
+		d.priority = 0
```

Comment:
Please use the constants defined in kv/kv.go

Reply:
Change with a constant